game-core/src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -273,7 +272,6 @@ public static void hostGame(final int port, final String playerName, final Strin
     commands.add("-D" + LOBBY_PORT + "="
         + messengers.getMessenger().getRemoteServerSocketAddress().getPort());
     commands.add("-D" + LOBBY_GAME_COMMENTS + "=" + comments);
-    commands.add("-D" + LOBBY_GAME_HOSTED_BY + "=" + messengers.getMessenger().getLocalNode().getName());
```

Comment:
Are you sure that `playerName` (used to initialize `TRIPLEA_NAME`) is identical to `messengers.getMessenger().getLocalNode().getName()` in all cases?  There's some weirdness that can go on when you have the same user logged in multiple times and an ordinal gets appended to the node name, but not necessarily the player name (or maybe that's vice versa).

Reply:
Can use `messengers.getMessenger().getLocalNode().getName()` instead: 3cf1e47dc

Given the 'hosted-by' and 'triplea-name' are asserted to be the same by the bot code, and the two values are then used interchangeably by bot code, that if there is any difference then we have a latent bug. I'm much rather for us to simplify and make any such problems more obvious and detectable, particularly when releases will be internal only for some time to come.

Reply:
Testing looked good. I'm reasonably confident on this one but it's hard to be 100% sure.